rbc/irtools.py

```diff
@@ -460,24 +558,246 @@ def cg_fflush(builder):
+exp2f exp2l exp10 exp10f exp10l log logf logl log2 log2f log2l log10
+log10f log10l logb logbf logbl ilogb ilogbf ilogbl pow powf powl sqrt
+sqrtf sqrtl cbrt cbrtf cbrtl hypot hypotf hypotl expm1 expm1f expm1l
+log1p log1pf log1pl clog clogf clogl clog10 clog10f clog10l csqrt
+csqrtf csqrtl cpow cpowf cpowl sinh sinhf sinhl cosh coshf coshl tanh
+tanhf tanhl csinh csinhf csinhl ccosh ccoshf ccoshl ctanh ctanhf
+ctanhl asinh asinhf asinhl acosh acoshf acoshl atanh atanhf atanhl
+casinh casinhf casinhl cacosh cacoshf cacoshl catanh catanhf catanhl
+erf erff erfl erfc erfcf erfcl lgamma lgammaf lgammal lgamma_r
+lgammaf_r lgammal_r gamma gammaf gammal j0 j0f j0l j1 j1f j1l jn jnf
+jnl y0 y0f y0l y1 y1f y1l yn ynf ynl rand srand rand_r random srandom
+initstate setstate random_r srandom_r initstate_r setstate_r drand48
+erand48 lrand48 nrand48 mrand48 jrand48 srand48 seed48 lcong48
+drand48_r erand48_r lrand48_r nrand48_r mrand48_r jrand48_r srand48_r
+seed48_r lcong48_r
+
+abs labs llabs fabs fabsf fabsl cabs cabsf cabsl frexp frexpf frexpl
+ldexp ldexpf ldexpl scalb scalbf scalbl scalbn scalbnf scalbnl
+significand significandf significandl ceil ceilf ceill floor floorf
+floorl trunc truncf truncl rint rintf rintl nearbyint nearbyintf
+nearbyintl round roundf roundl roundeven roundevenf roundevenl lrint
+lrintf lrintl lround lroundf lroundl llround llroundf llroundl fromfp
+fromfpf fromfpl ufromfp ufromfpf ufromfpl fromfpx fromfpxf fromfpxl
+ufromfpx ufromfpxf ufromfpxl modf modff modfl fmod fmodf fmodl
+remainder remainderf remainderl drem dremf dreml
+
+copysign copysignf copysignl signbit signbitf signbitl nextafter
+nextafterf nextafterl nexttoward nexttowardf nexttowardl nextup
+nextupf nextupl nextdown nextdownf nextdownl nan nanf nanl
+canonicalize canonicalizef canonicalizel getpayload getpayloadf
+getpayloadl setpayload setpayloadf setpayloadl setpayloadsig
+setpayloadsigf setpayloadsigl isgreater isgreaterequal isless
+islessequal islessgreater isunordered iseqsig totalorder totalorderf
+totalorderl totalordermag totalorderf totalorderl fmin fminf fminl
+fmax fmaxf fmaxl fminmag fminmagf fminmagl fmaxmag fmaxmagf fmaxmagl
+fdim fdimf fdiml fma fmaf fmal fadd faddf faddl fsub fsubf fsubl fmul
+fmulf fmull fdiv fdivf fdivl '''.strip().split())
+
+
+def get_stdio_functions():
+    """Return all stdio library function names.
+
+    Reference: http://www.cplusplus.com/reference/cstdio/
+    """
+    return list(''' remove rename tmpfile tmpnam fclose fflush fopen freopen setbuf
+setvbuf fprintf fscanf printf scanf snprintf sprintf sscanf vfprintf
+vfscanf vprintf vscanf vsnprintf vsprintf vsscanf fgetc fgets fputc
+fputs getc getchar gets putc putchar puts ungetc fread fwrite fgetpos
+fseek fsetpos ftell rewind clearerr feof ferror perror '''.strip().split())
+
+
+def get_stdlib_functions():
+    """Return all stdlib library function names.
+
+    Reference: http://www.cplusplus.com/reference/cstdlib/
+    """
```

Comment:
Same here!

https://github.com/xnd-project/rbc/blob/9766f89f89e4be788c23b43f8fb8fc1276b5346f/rbc/irtools.py#L57

Reply:
Removed in favor of `libfuncs.Stdlib()` instance.